.env.example

```
@@ -9,28 +9,47 @@ ENABLE_PRIVATE_APP_DOWNLOADING=false
 ENABLE_TELEMETRY=false
 
 # =============================================================================
-# MINIO (local or cloud S3-compatible storage)
+# GARAGE (recommended for local development via AWS SDK)
 # =============================================================================
-STORAGE_DRIVER=minio
-MINIO_SECURE=false
+# These credentials are imported automatically by the Docker Compose Garage bootstrap.
+STORAGE_DRIVER=aws
+S3_ACCESS_KEY=GK8fcbbed327a9ff97f250eb4e
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The S3_ACCESS_KEY key should go before the STORAGE_DRIVER key

(UnorderedKey)